src/store/local-storage.ts

```diff
@@ -6,6 +6,7 @@ export const loadState = () => {
         if (serializedState === null) {
             return undefined;
         }
+        console.log("State loaded from local storage");
```

Comment:
Are these console.logs intended to be kept?

Reply:
ref https://github.com/Project-Lavinia/Lavinia-client/pull/57#discussion_r240321394